runner/templates/configmap.yaml

```diff
@@ -22,12 +22,42 @@ data:
       name = "{{ template "fullname" . }}"
       url = {{ .Values.gitlabUrl | quote }}
       token = {{ .Values.token | quote }}
-      executor = {{ default "docker" .Values.executor.type | quote}}
+      executor = {{ default "docker" .Values.executor.type | quote }}
```

Comment:
I thought about moving the docker executor configuration to `Values.executor.docker` like I did with the kubernetes, but that would break configurations that work today.

Maybe I can change the check from `if .Values.executor.kubernetes` to `if eq .Values.executor.type "kubernetes"`. 
This way it won't break if `values.executor.kubernetes` is uncommented with a `docker` executor type.